### ISSUE TITLE
Auto Generated JavaDoc

### ISSUE THREAD

**Issue by Toilal** [hazendaz/javabean-tester#12]
It's an open discussion, but I don't think auto generated JavaDoc is a good idea.

It make source files less readable and doesn't add any value.

Could we remove the auto generated stuff for the next version ?

**Comment by hazendaz** [comment]
I tend to agree with you that JavaDoc in general isn't necessary if auto generated but it does tend to shut up various scanners.  I was using jautodoc as more of a jumping off point.  While I'm not totally against just pulling out the fluff, let me think about it some more.

**Comment by hazendaz** [comment]
I think the usefulness per various scanners is more important than removing these.  However, I think these are just a baseline and what we need to do is to enhance the javadocs.  Therefore, I'm leaving this open but relabeling it to enhancement needs.

**Comment by hazendaz** [comment]
closing as decision is to keep javadocs, any improvements can be done as needed and many are expanded now.
